src/game_io/game_class_packet.cc

```diff
@@ -43,7 +43,7 @@ void GameClassPacket::read(FileSystem& fs, Game& game, MapObjectLoader* /* mol *
 
 			game.list_of_scenarios_.clear();
 			if (packet_version >= 6) {
-				for (size_t i = fr.unsigned_32(); i; --i) {
+				for (size_t i = fr.unsigned_32(); i != 0u; --i) {
```

Comment:
```suggestion
				for (size_t i = fr.unsigned_32(); i > 0u; --i) {
```
I know I'm a pain in the... :)

Reply:
Not a problem :) I also find `> 0` more readable than `!= 0` but I missed a lot of them when going through clang-tidy's autofix results